### PR TITLE
CLC-5095 - bCourses CSS: Site Find a Person to Add styling

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.ets.berkeley.edu/jira/browse/CLC-5095

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js [hunk 1]
```diff
@@ -69,6 +69,14 @@
       return false;
     };
 
+    var scrollToTop = function() {
```
Comment (christianv) [inline src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js]: @redconfetti seems like this is a copy / paste from `canvasCourseGradeExportController` - would it be possible to extract this `scrollToTop` function into the util service?

https://github.com/ets-berkeley-edu/calcentral/blob/26634eb8a96a6fa604cbef9e8383519cc30449c3/src/assets/javascripts/angular/controllers/pages/canvasCourseGradeExportController.js#L135

File: src/assets/stylesheets/_canvas_base.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@ $bc-base-font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
 
 %bc-form-input-select {
   @extend %bc-canvas-base-typography;
+  -webkit-appearance: none;
```
Comment (christianv) [inline src/assets/stylesheets/_canvas_base.scss]: @redconfetti you probably also need `-moz-appearance: none;`

File: src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js (hunk 1 above)
Comment (redconfetti) [inline src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js]: @christianv It relies on $window. Would it be possible to inject $window into the utilService just for this one function?

Comment (christianv) [inline src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js]: @redconfetti afaik `$window` should already be part of the `utilService`:
https://github.com/ets-berkeley-edu/calcentral/blob/26634eb8a96a6fa604cbef9e8383519cc30449c3/src/assets/javascripts/angular/services/utilService.js#L4

Comment (redconfetti) [inline src/assets/javascripts/angular/controllers/pages/canvasCourseAddUserController.js]: @christianv This is fixed.

File: src/assets/stylesheets/_canvas_base.scss (hunk 2 above)
Comment (redconfetti) [inline src/assets/stylesheets/_canvas_base.scss]: @christianv Done

File: src/assets/javascripts/angular/services/utilService.js [hunk 3]
```diff
@@ -126,11 +126,14 @@
...
 
     /**
-     * Send a message triggering the parent page to scroll to the top
+     * Send a message triggering the parent page to scroll to the top.
+     * Assumes iframe environment only applies to bCourses / Canvas LTI
      */
     var iframeScrollToTop = function() {
       if (isInIframe) {
         iframePostMessage(JSON.stringify({subject: 'changeParent', scrollToTop: true}));
+      } else {
```
Comment (raydavis) [inline src/assets/javascripts/angular/services/utilService.js]: Nice improvement!

**Pull request merged**